src/components/Topics/TopicViews.tsx

```diff
@@ -5,6 +5,7 @@ import { RootState } from '../../store/reducers';
 import { ResourcesSlice } from '../../store/resourcesSlice';
 import TopicHeader from './TopicHeader';
 import { Typography } from '@material-ui/core';
+import theme from '../../CustomStyles';
```

Comment:
in this -- i think the design isn't matching the figma -- i think the "Resources" title and "Facing Trouble" title are both removed now ! (added latest design into notion as well)